.travis.yml

```diff
@@ -2,10 +2,26 @@ language: python
 python:
   - "2.7"
 
+sudo: required
+
 install:
   - pip install pep8
 
+services:
+  - docker
+
 script:
   # Run pep8 on all .py files in all subfolders
   # ignore E402: module level import not at top of file
   - find . -name \*.py -exec pep8 --ignore=E402 {} +
+  - sudo docker run -d -v ceph:/etc/ceph -e MON_IP=172.17.0.2 -e CEPH_PUBLIC_NETWORK=172.17.0.0/24 --name ceph ceph/demo
+  - sleep 30
+  - sudo docker run -d --name hil chemistrysourabh/hil
+  - sudo docker pull chemistrysourabh/bmi-ci
```

Comment:
you mean the repo names ? Ya that is part of my agenda. I didnt knew docker would use my Docker ID as my repo name. Will change it.